File: csharp/DesiArchitect.DesignPatterns/07_Facade/After/NotificationService.cs
namespace DesiArchitect.DesignPatterns.Facade.After;

public class NotificationService : INotificationService
{
    public void Send(string phone, string pnr)
        => Console.WriteLine($"  [After] NotificationService: SMS sent to {phone} - PNR: {pnr}");
}

File: typescript/src/07-facade/after/notification-service.ts
import type { INotificationService } from './i-notification-service.js';

export class NotificationService implements INotificationService {
  send(phone: string, pnr: string): void {
    console.log(`  [After] NotificationService: SMS sent to ${phone} - PNR: ${pnr}`);
  }
}